frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/StrategyExecution/StrategyExecution.tsx

```diff
@@ -1,372 +1,110 @@
-                                    <>
-                                        {isSetTo}
-                                        <StringTruncator
-                                            maxWidth='300'
-                                            text={value}
-                                            maxLength={50}
-                                        />
-                                    </>
-                                }
-                            />
-                        </StyledValueContainer>
-                    ) : null;
-                }
-
-                case 'number': {
-                    const number = parseParameterNumber(parameters[name]);
-                    return parameters[name] !== '' && number !== undefined ? (
-                        <StyledValueContainer>
-                            {nameItem}
-                            {isSetTo}
-                            <StringTruncator
-                                maxWidth='300'
-                                text={String(number)}
-                                maxLength={50}
-                            />
-                        </StyledValueContainer>
-                    ) : null;
-                }
-                case 'default':
-                    return null;
-            }
-
-            return null;
-        });
-    }, [parameters, definition]);
-
-    if (!parameters) {
-        return <NoItems />;
-    }
-
-    const listItems = [
-        strategySegments && strategySegments.length > 0 && (
-            <FeatureOverviewSegment
-                segments={strategySegments}
-                disabled={strategy.disabled}
-            />
-        ),
-        constraints.length > 0 && (
-            <ConstraintAccordionList
-                constraints={constraints}
-                showLabel={false}
-            />
-        ),
-        strategy.name === 'default' && (
-            <>
-                <StyledValueContainer sx={{ width: '100%' }}>
-                    The standard strategy is <Badge color='success'>ON</Badge>{' '}
-                    for all users.
-                </StyledValueContainer>
-            </>
-        ),
-        ...(parametersList ?? []),
-        ...(customStrategyList ?? []),
-    ].filter(Boolean);
+    const { isCustomStrategy, customStrategyParameters: customStrategyItems } =
+        useCustomStrategyParameters(strategy, strategies);
+    const strategyParameters = useStrategyParameters(strategy, displayGroupId);
+    const { constraints } = strategy;
+    const strategySegments = segments?.filter((segment) =>
+        strategy.segments?.includes(segment.id),
+    );
 
     return (
-        <>
-            <ConditionallyRender
-                condition={
-                    !BuiltInStrategies.includes(strategy.name || 'default')
-                }
-                show={<CustomStrategyDeprecationWarning />}
-            />
+        <StyledGrayscale enabled={strategy.disabled === true}>
```

Comment:
Oh, this one is kinda funny. Maybe we can make the prop more descriptive? It's hard to know what enable means. How about something like this? 🤷🏼 

```suggestion
        <FilterContainer addGrayscaleFilter={strategy.disabled}>
```